examples/ruby/mutual_auth/client_certs/trusted_server_roots.pem

```diff
@@ -0,0 +1,15 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Consolidated server_certs/trusted_client_roots.pem and client_certs/ca.pem into one file.